### PR TITLE
Add Play 1.3.0

### PR DESCRIPTION
**Pull request by ghostganz**
Adding the "new" stable Play 1.3.0 as "play13"

### CODE REVIEW

File: play13.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require 'formula'
```
Comment (DomT4) [inline play13.rb]: This line can be snipped.

File: play13.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+class Play13 < Formula
+  homepage 'http://www.playframework.org/'
```
Comment (DomT4) [inline play13.rb]: Double quotes everywhere please, Thanks!

**Comment by ghostganz** [comment]
@DomT4 Thanks, I've fixed the quotes and the require now.

**Comment by DomT4** [comment]
Thanks! Any chance of a small test? See cmake.rb for an example of an application formula with a good test and tinyxml2.rb for an example of a library formula with a good test. Thanks!

**Comment by ghostganz** [comment]
I only ported it from the play 1.2.x recipe, I'll leave it for someone with more spare time to create a test. :)

**Pull request closed without merging**